src/core/menu_items/WifiMenu.cpp

```diff
@@ -9,7 +9,7 @@
 #include "modules/wifi/sniffer.h"
 #include "modules/wifi/wardriving.h"
 #include "modules/wifi/wifi_atks.h"
-#include "modules/ble/ble_jammer.h"
+#include "modules/NRF24/nrf_jammer.h"
```

Comment:
I think this line can be removed since you removed the jammer from the options menu